projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item-update/index.js

```diff
@@ -69,8 +81,30 @@ export function useSyncMedia( { id, title, description } ) {
 		}
 
 		updateMedia( dataToUpdate ).then( () => updateInitialState( { title, description } ) );
+
+		// Video Chapters //
+		if ( ! dataToUpdate?.description?.length ) {
+			return;
+		}
+
+		// Upload .vtt file if it description contains chapters
+		const chapters = extractVideoChapters( dataToUpdate.description );
+		if ( ! chapters?.length ) {
+			return;
+		}
```

Comment:
`generateChaptersFile` already checks this. You could use something like:

```js
const vttFile generateChaptersFile( dataToUpdate.description );
if ( ! vttFile ) return; // vttFile === null
```

Reply:
https://github.com/Automattic/jetpack/issues/26268